### DISCUSSION TITLE
Platform to Business Notice

### DISCUSSION THREAD

**Discussion by MattiSG** [OpenTermsArchive/terms-types#18] [Added]
### SkyScanner Regulation 2019/1150 – Platform to Business Regulation

https://www.skyscanner.net/media/regulation-2019-1150-platform-to-business-regulation

### Facebook Platform to Business Notice

https://www.facebook.com/legal/PlatformtoBusinessNotice

### Etsy European Union Platform-to-Business Regulation

https://help.etsy.com/hc/en-gb/articles/360051541413-European-Union-Platform-to-Business-Regulation-on-Etsy

**Comment by MattiSG** [comment]
### Recap table of provided examples

| URL                                                                                                                                | Title                                                  | Service    | Contains                                                                                                  | Audience       |   |
|------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------|------------|-----------------------------------------------------------------------------------------------------------|----------------|---|
| [URL](https://www.skyscanner.net/media/regulation-2019-1150-platform-to-business-regulation)                                       | Regulation 2019/1150 – Platform to Business Regulation | Skyscanner | List of relevant terms — Ranking parameters — Partner rating parameters — Complaints handling — Mediation | Partners       |   |
| [URL](https://www.facebook.com/legal/PlatformtoBusinessNotice)                                                                     | Platform to Business Notice                            | Facebook   | Ranking parameters — Complaints handling — Mediation — Transparency                                       | Business users |   |
| [URL](https://help.etsy.com/hc/en-gb/articles/360051541413-European-Union-Platform-to-Business-Regulation-on-Etsy?segment=selling) | European Union Platform-to-Business Regulation         | Etsy       | List of relevant terms — Transparency — Complaints handling — Mediation                                   | Sellers        |   |

### Relevance

These documents consolidate information mandated by the EU P2B Regulation for intermediation platforms. While the regulation does not mandate that they are all found in a single document, it seems several large actors have decided (for the moment) to implement it this way. This might evolve over time, either by spreading each item on a separate page, akin to Community Guidelines (in which case multipage support would be sufficient), or by spreading the information in the most relevant higher-order document. For example, mediation information for partners could be part of a larger Complaints Policy, transparency could be part of a larger Transparency Report, etc.

Since these documents have only been mandated since July 2020, it is too early to know for sure how the industry will implement this regulation more generally (see https://github.com/ambanum/OpenTermsArchive/issues/773 for some background on how it took years for the presentation of Community Guidelines to converge towards multiple pages).

Some existing and potential document types (see below) seem to be a subset of what could be a `P2B Notice`. This document type would thus be used for industry implementations that specifically consolidate P2B information, and as a default for all mixed cases. More specific P2B-mandated information could be tracked as independent documents when providers implement them as dedicated pages.
The alternative would be to track these more specific documents as multipages of `P2B Notice`, thus decreasing the number of document types, but also decreasing specificity and cross-platform comparability.

### Competing document types


#### `Ranking Parameters Description`

This document type contains a subset of the information mandated by the P2B regulation, which can be found in 2 out of 3 examples. The third example, Etsy, has both a potential `P2B Notice` **and** a [dedicated `Ranking Parameters Description` page](https://www.etsy.com/legal/policy/search-advertisement-ranking-disclosures/899478564529).

This reminds of many other cases where document types can be more specific when the provider separates them, but also integrated in the wider one, the most emblematic case being `Privacy Policy` sometimes part of `Terms of Use` for the smallest providers.

#### `Complaints Policy`

Similar to `Ranking Parameters Description`, but even wider: complaints handling is not specifically mandated by P2B regulation, so the integration of those elements in the `P2B Notice` document could compete with an existing `Complaints Policy`, and the line between the two could be blurry.

The industry seems to evolve in reaction to the P2B regulation by creating a dedicated Mediation / Complaints document for business users, as outlined in https://github.com/ambanum/OpenTermsArchive/discussions/933.

Once again, this leads to consider the `P2B Notice` potential document type as a superset of both `Ranking Parameters Description` and a potential `Business Mediation Policy`.

### Tryptich

Writer: search engine or intermediation service provider
Audience: business user
Object: complaints handling, mediation, transparency and other rights and information mandated by the P2B regulation
References:
- Open Terms Archive Discussion: https://github.com/ambanum/OpenTermsArchive/discussions/940
- P2B Regulation 2019/1150: https://eur-lex.europa.eu/eli/reg/2019/1150/oj

### Naming

#### Intent

- Considering the wording throughout the regulation (“_Regulation (EU) 2019/1150 of the European Parliament and of the Council of 20 June 2019 on promoting fairness and transparency for **business users** of online intermediation services_”), I suggest to use only “business users” and not “commercial users” as used in some other document types.
- These documents are not imperative and describe, at the most, rights (and not duties). They thus are a “notice” or “information”, but do not constitute a “policy” or “conditions”.
- These documents only apply to platforms (as defined in the P2B regulation), and that should be made clear in the name.

#### Proposals

1. P2B Notice
2. Platform-to-Business Notice
3. Platform-to-Business Information
4. Platform Business Users Notice
5. Platform Business Users Rights
6. Platform to Business Users Service Notice
7. Platform to Business Users Notice
8. Business Users Notice

**Reply by lucielechardoy** [reply]
Thanks @MattiSG  for the recap. I would vote for 2. Platform-to-Business Notice (thus spelling out P2B in case the acronym is not obvious)

**Reply by martinratinaud** [reply]
2 also

**Reply by clementbiron** [reply]
2 is my favorite too.

Why `Platform-to-Business Notice` instead of `Platform to Business Notice`. Did I miss anything?

**Reply by MattiSG** [reply]
@clementbiron thanks for pointing this out. I don't see a good reason anymore to add hyphens: there seems to be no dedicated short name for the regulation, and none of the given example documents have a dash in their title. Let's go with `Plaform to Business Notice`! 👍

**Comment by MattiSG** [comment]
Added via https://github.com/ambanum/OpenTermsArchive/pull/941!
